src/Files.App/Views/Pages/Properties.xaml

```diff
@@ -2,174 +2,182 @@
+									<Viewbox
+										Width="24"
+										Height="24"
+										Margin="0,4,0,0"
+										Visibility="{x:Bind IsSelected, Converter={StaticResource ObjectToBoolConverter}, ConverterParameter=Invert, Mode=OneWay}">
+										<PathIcon
+											x:Name="NavigationViewItemOutlinePathIcon"
+											HorizontalAlignment="Center"
+											Data="{x:Bind OutlinePathIcon}" />
+									</Viewbox>
+
+									<!--  Filled Glyph  -->
+									<Viewbox
+										Width="24"
+										Height="24"
+										Margin="0,0,0,4"
+										Visibility="{x:Bind IsSelected, Converter={StaticResource ObjectToBoolConverter}, Mode=OneWay}">
+										<PathIcon
+											x:Name="NavigationViewItemFilledPathIcon"
+											HorizontalAlignment="Center"
+											Data="{x:Bind FilledPathIcon}"
+											Foreground="{ThemeResource AccentFillColorDefaultBrush}" />
+									</Viewbox>
+								</Grid>
+
+								<!--  Label  -->
+								<TextBlock
+									x:Name="NavigationViewItemNameTextBlock"
+									Grid.Row="1"
+									Margin="-2,0,-4,0"
+									HorizontalAlignment="Center"
+									x:Load="{x:Bind IsSelected, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}"
+									FontSize="10"
+									Style="{StaticResource CaptionTextBlockStyle}"
+									Text="{x:Bind Name}" />
+
+							</StackPanel>
+						</NavigationViewItem.Content>
 					</NavigationViewItem>
-				</NavigationView.MenuItems>
-			</NavigationView>
-		</Border>
+				</DataTemplate>
+			</NavigationView.MenuItemTemplate>
 
-		<ScrollViewer Grid.Row="1">
-			<Frame x:Name="contentFrame" IsNavigationStackEnabled="False" />
-		</ScrollViewer>
+		</NavigationView>
 
+		<!--  NavigationView Content, Save/Cancel buttons  -->
 		<Grid
-			Grid.Row="2"
-			Padding="12"
-			Background="{ThemeResource CardBackgroundFillColorSecondary}"
+			Grid.Row="1"
+			Grid.Column="1"
+			Background="{ThemeResource LayerFillColorDefaultBrush}"
 			BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
-			BorderThickness="0,1,0,0">
-			<Grid.ColumnDefinitions>
-				<ColumnDefinition Width="*" />
-				<ColumnDefinition Width="*" />
-				<ColumnDefinition Width="Auto" />
-			</Grid.ColumnDefinitions>
-			<Button
-				x:Name="OKButton"
-				Grid.Column="1"
-				MinWidth="100"
-				Margin="0,0,10,0"
-				HorizontalAlignment="Right"
-				Click="OKButton_Click"
-				Content="{helpers:ResourceString Name=PropertiesDialogOKButton/Content}"
-				Style="{StaticResource AccentButtonStyle}" />
-			<Button
-				x:Name="CancelButton"
-				Grid.Column="2"
-				MinWidth="100"
-				HorizontalAlignment="Right"
-				Click="CancelButton_Click"
-				Content="{helpers:ResourceString Name=Cancel}" />
+			BorderThickness="1,1,0,0"
+			CornerRadius="8,0,0,0">
+			<Grid.RowDefinitions>
+				<RowDefinition Height="*" />
+				<RowDefinition Height="Auto" />
+			</Grid.RowDefinitions>
+
+			<Frame x:Name="contentFrame" IsNavigationStackEnabled="False" />
+
+			<Grid
+				Grid.Row="2"
+				Height="58"
```

Comment:
Height should be 60 (evenly divisible by 4 rule)

Reply:
If do so, it doesn't looks familiar with the MS Store